src/main/scala/io/iohk/ethereum/jsonrpc/JsonMethodsImplicits.scala

```diff
@@ -38,7 +38,7 @@ trait JsonMethodsImplicits {
     }.toEither.left.map(_ => InvalidParams())
 
   private def decode(s: String): Either[JsonRpcError, Array[Byte]] = {
-    if(!s.startsWith("0x")) Left(InvalidParams())
+    if(!s.isEmpty && !s.startsWith("0x")) Left(InvalidParams())
```

Comment:
So I noticed you made this change:

[Data should be prefixed with 0x](https://github.com/input-output-hk/etc-client/commit/8b76fd2a6f7e854b00bb1137d446f45db74ef19c)

No, it should not. We're getting hit all the time by following the documentation to the word, and this PR is a perfect example ;)
Geth does not require bytes to be prefixed with `0x`, which is why I changed that in #195. Could you change it back, please?